2024-09-20 17:25:43,393 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Current SDK version is 0.18.0
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Configure stats pid to 4560
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Loading settings from C:\Users\LWJ\.config\wandb\settings
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Loading settings from F:\Repository\rl_algorithm\wandb\settings
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'ppo_main.py', 'program_abspath': 'f:\\Repository\\rl_algorithm\\ppo_main.py', 'program': 'f:\\Repository\\rl_algorithm\\ppo_main.py'}
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:_log_setup():525] Logging user logs to F:\Repository\rl_algorithm\wandb\run-20240920_172543-q2bzdu30\logs\debug.log
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:_log_setup():526] Logging internal logs to F:\Repository\rl_algorithm\wandb\run-20240920_172543-q2bzdu30\logs\debug-internal.log
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:init():609] calling init triggers
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:init():616] wandb.init called with sweep_config: {}
config: {}
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:init():659] starting backend
2024-09-20 17:25:43,394 INFO    MainThread:4560 [wandb_init.py:init():663] setting up manager
2024-09-20 17:25:43,396 INFO    MainThread:4560 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-09-20 17:25:43,398 INFO    MainThread:4560 [wandb_init.py:init():671] backend started and connected
2024-09-20 17:25:43,401 INFO    MainThread:4560 [wandb_init.py:init():766] updated telemetry
2024-09-20 17:25:43,604 INFO    MainThread:4560 [wandb_init.py:init():799] communicating run to backend with 90.0 second timeout
2024-09-20 17:27:14,340 ERROR   MainThread:4560 [wandb_init.py:init():825] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-20 17:27:14,340 ERROR   MainThread:4560 [wandb_init.py:init():1244] error in wandb.init()
Traceback (most recent call last):
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\wandb_init.py", line 1234, in init
    return wi.init()
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\wandb_init.py", line 831, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-20 17:27:15,253 WARNING MsgRouterThr:4560 [router.py:message_loop():77] message_loop has been closed
